cordformation/src/main/kotlin/net/corda/plugins/Baseform.kt

```diff
@@ -143,11 +145,11 @@ open class Baseform : DefaultTask() {
         val networkBootstrapperClass = loadNetworkBootstrapperClass()
         val networkBootstrapper = networkBootstrapperClass.newInstance()
         val bootstrapMethod = networkBootstrapperClass.getMethod("bootstrap", Path::class.java, List::class.java).apply { isAccessible = true }
-        // Call NetworkBootstrapper.bootstrap
+        // Create a list of all cordapps used in this network and pass it to the bootstrapper.
```

Comment:
This comment is too much of a what, not why. The below comment is useful because it provides a way to mentally shortcut trying to read reflection code but this comment isn't.